CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v8.2.1](https://github.com/puppetlabs/puppetlabs-apache/tree/v8.2.1) (2022-09-26)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-apache/compare/v8.2.0...v8.2.1)
+
+### Fixed
+
+- \(MAINT\) Removing support for CentOS 8 [\#2314](https://github.com/puppetlabs/puppetlabs-apache/pull/2314) ([jordanbreen28](https://github.com/jordanbreen28))
```

Comment:
This doesn't sound like a bugfix.

Reply:
Ah, this was reverted.
We just missed swaping it to a amaintanence label